src/state/is-within-visible-bounds-of-droppable.js

```diff
@@ -55,7 +55,8 @@ export const isDraggableWithin = (bounds: Spacing) => {
   const isWithinVertical = isWithin(top - 1, bottom + 1);
 
   return (draggable: DraggableDimension): boolean => {
-    const fragment: DimensionFragment = draggable.page.withMargin;
+    // margin may bleed outside of container size
+    const fragment: DimensionFragment = draggable.page.withoutMargin;
```

Comment:
swapped to `withoutMargin` to be more lineant. I found a scenario where a draggables margin bleeds over the droppable thanks to margin collapsing